### PR TITLE
Assignment2b done

### PR DESCRIPTION
**Pull request by polkiko**

### CODE REVIEW

File: Assignment2/polkiko-fromRDFtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "items": "rdfs:Class",
+    "data": "rdfs:label"
+  },
+  "items": [
+    {
+      "@type": "rdfs:Class",
+      "@id": "http://www.web.com/ontologies/Class01",
+      "vocab:includes": [
+        {
+          "@type": "rdfs:Class",
```
Comment (ocorcho) [inline Assignment2/polkiko-fromRDFtoJSON-LD.jsonld:15]: This should not be the type of this object. It should be something different.

File: Assignment2/polkiko-fromRDFtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          "@id": "http://www.web.com/ontologies/Sensor029"
+        },
+        {
+          "@type": "rdfs:Class",
```
Comment (ocorcho) [inline Assignment2/polkiko-fromRDFtoJSON-LD.jsonld:19]: Same here, it should be a Computer class

**Review by ocorcho — CHANGES_REQUESTED** [review]
Some examples provided

**Review by polkiko — COMMENTED** [review]
Change in classes

File: Assignment2/polkiko-fromRDFtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          "@type": "rdfs:Computer",
+          "@id": "http://www.web.com/ontologies/Computer101"
+        }
+      ]
+    },
+    {
+      "@type": "rdfs:Sensor",
+      "@id": "http://www.web.com/ontologies/Sensor029",
+      "vocab:hasMeasurement": {
+        "@type": "rdfs:Measurement",
```
Comment (ocorcho) [inline Assignment2/polkiko-fromRDFtoJSON-LD.jsonld:28]: Should not be rdfs:Measurement, since it is not describe in RDF Schema, but I accept it.

**Review by ocorcho — APPROVED** [review]

**Pull request merged**